Self-Certification/Questionnaire/ISO5230-2020/en/OpenChain Self-Certification Questionnaire 2021-11-26.md

```diff
@@ -12,20 +12,19 @@ OpenChain ISO/IEC 5230:2020 is the International Standard for open source licens
 
 You can adopt OpenChain ISO/IEC 5230:2020 by self-certification in your own time or working with a service provider for independent assessment or third-party certification. Our recommended path is self-certification and we provide this questionnaire to support this with a series of "yes" or "no" questions.
 
-We have a lot of resources to support you if you need assistance. You can join our mailing lists, our webinars, our group calls and our regional work groups to discuss challenges with your peers and in your native language. You can get started here:
+We have a lot of resources to support you if you need assistance. You can join our mailing lists, our webinars, our group calls and our regional work groups to discuss challenges with your peers and in your native language. You can get started here:<br>
+https://www.openchainproject.org/community
 
-[[https://www.openchainproject.org/community]{.underline}](https://www.openchainproject.org/community)
+As part of our online support you can also self-certify using our web app for free here:<br>
+https://certification.openchainproject.org
```

Comment:
Recommend enclose in angle brackets to follow markdown guidelines.
https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses